src/sentry/ingest/inbound_filters.py

```diff
@@ -217,7 +219,19 @@ def _get_filter_settings(project_config, flt):
 
 
 class _LegacyBrowserFilterSerializer(_FilterSerializer):
-    subfilters = MultipleChoiceField(
+    subfilters = serializers.MultipleChoiceField(
```

Comment:
Switched from custom `MultipleChoiceField` that was made a long time ago to built-in drf-rest MultipleChoiceField